images/router/haproxy/conf/haproxy-config.template

```diff
@@ -58,7 +58,7 @@ listen stats :1936
 {{ end }}
 
 frontend public
-  bind :80
+  bind :{{getEnv "ROUTER_SERVICE_HTTP_PORT" "80"}}
```

Comment:
Reading env in the template is a bit questionable - there may be some security implications in the future.  However, it's fine for now.